rllib/algorithms/algorithm.py

```diff
@@ -2506,15 +2505,15 @@ def __getstate__(self) -> dict:
 
     def __setstate__(self, state: dict):
         if hasattr(self, "workers") and "worker" in state:
-            self.workers.local_worker().restore(state["worker"])
+            self.workers.local_worker().set_state(state["worker"])
```

Comment:
Align with `Policy.get_state/set_state` API.
Soft-deprecate for now. These methods are not really used by users, more by RLlib internally.

Reply:
Also to emphasize that we are NOT reading from or writing to a file, but rather return a state dict (or use one to set some state).